prow/cluster/starter.yaml

```diff
@@ -662,3 +709,35 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: "tide"
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: default
+  name: "statusreconciler"
```

Comment:
Does every service really needs it's own service account as opposed to one scoped for prow or some larger subset thereof?

Reply:
More granular permissions are better IMO